app/jobs/send_welcome_email.rb

```diff
@@ -1,5 +1,6 @@
 class SendWelcomeEmail < Struct.new(:person_id, :community_id)
 
+  include SessionContextSerializer
```

Comment:
First Q: My thinking was to include it to every job, so that if someone writes some code that depends on the SessionContextStore content (e.g. add a new API call), there's no need to think whether the SessionContextStore is initialized or not. Does this make sense?

Second Q: Yes it should. But let me test that once more.